app/graph/types/project_media_type.rb

```diff
@@ -284,5 +284,13 @@
     }
   end
 
+  connection :suggested_similar_items, -> { ProjectMediaType.connection_type } do
+    resolve -> (project_media, args, _ctx) {
```

Comment:
We're not using `args` in this method, so please rename it to `_args` (convention to mark parameters that are not used)

Reply:
Is that what CodeClimate is going off about?

Reply:
Yes! :D